<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppointmentWindow</class>
 <widget class="QMainWindow" name="AppointmentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="DisplayWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>201</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/img/Customer Copy.png);</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>姓名:</string>
    </property>
   </widget>
   <widget class="QLabel" name="majorLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>专业:</string>
    </property>
   </widget>
   <widget class="QLabel" name="workTime">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>200</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>从业时间:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>431</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>个人简介</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTextEdit" name="briefIntroduceTextEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>260</y>
      <width>341</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="commitButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>500</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>提交</string>
    </property>
   </widget>
   <widget class="QLabel" name="appointmentTime">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>500</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>预约时间:</string>
    </property>
   </widget>
   <widget class="QLabel" name="dateLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>500</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>892</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DisplayWidget</class>
   <extends>QWidget</extends>
   <header>displaywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
